joomla/joomla-cms
# [4.0] Update RandomImageHelper.php

Fix for #22065 and improved code to scaled to the specified final height or width (so that the final height or width is not exceeded and there is no loss in dimension) why wrong complex, If it's that simple ;)

Pull Request for Issue #22065 .

### Testing Instructions
See steps: #22065
Try once with custom height, once with custom width.
Once with both and once with empty height and width.

### Expected result
Final height or width should be calculated in dimension even if no width or hight is set
Check px specification in width or heigt attibute (Not the given look, if huge px, because of box size limits possible ;)

### Actual result
Works a bit complex  without dimenson loss but it works only if width or hight is set!

### Documentation Changes Required
No
